app/views/green_lanes/check_your_answers/show.html.erb

```diff
@@ -22,7 +22,7 @@
       %>
     <% end %>
 
-    <% if all_exemptions_met?(1, @category_one_assessments, @answers) %>
+    <% if all_exemptions_met?(1, @category_one_assessments, @answers) && !@category_two_assessments_without_exemptions %>
```

Comment:
We need to show this box when there are cat 2 assessments without exemptions not just when all assessments are met in cat 1